ads/google/imaVideo.js

```diff
@@ -1244,10 +1282,11 @@ function postMessage(data) {
  * @visibleForTesting
  */
 export function getPropertiesForTesting() {
-  return {adContainerDiv, adRequestFailed, adsActive, adsManagerWidthOnLoad,
+  return {adContainerDiv, allAdsCompleted,
```

Comment:
Oof :)

Not your fault, but would you mind setting one item per line?

Reply:
Definitely 😄